api/db/migrations/20200515163806_add-column-createdat-updatedat-to-memberships.js

```diff
@@ -0,0 +1,19 @@
+const TABLE_NAME = 'memberships';
+const CREATED_AT_COLUMN = 'createdAt';
+const UPDATED_AT_COLUMN = 'updatedAt';
+
+exports.up = async function (knex){
+
+    await knex.schema.table(TABLE_NAME, (table) => {
+      table.dateTime(CREATED_AT_COLUMN).defaultTo(knex.fn.now());
+      table.dateTime(UPDATED_AT_COLUMN).defaultTo(knex.fn.now());
```

Comment:
D'après mes petits tests en local, écrit comme ça, ça va mettre à la date de maintenant les nouvelles colonnes sur les lignes existantes.

Personnellement ça ne me choque pas (mais à ce moment-là, autant marquer les colonnes `NOT NULL` comme les autres !). Si vous voulez absolument que les anciennes lignes aient `NULL`, il faut créer les colonnes sans valeur par défaut puis changer leur valeur par défaut à `now()`.

Reply:
Bien vu, on a opté pour le NULL comme valeur par défault et j'ai bien le comportement attendu avec ta proposition. Thanks a lot